go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/cloudposse/github-status-updater
 
-go 1.13
+go 1.17
+
+require github.com/google/go-github/v42 v42.0.0
 
 require (
-	github.com/google/go-github v17.0.0+incompatible
-	github.com/google/go-querystring v1.0.0 // indirect
-	golang.org/x/net v0.0.0-20200625001655-4c5254603344
+	github.com/google/go-querystring v1.1.0 // indirect
```

Comment:
can use one `require`

also please update https://github.com/cloudposse/github-status-updater/blob/master/LICENSE#L189

Reply:
The format of `go.mod` changed as of `go` v1.17. See https://go.dev/ref/mod#graph-pruning

License updated.